frontend/src/concepts/areas/const.ts

```diff
@@ -178,6 +179,11 @@ export const SupportedAreasStateMap: SupportedAreasState = {
     reliantAreas: [SupportedArea.MODEL_SERVING],
     //TODO: Add Llama Stack component when details known.
   },
+  [SupportedArea.LM_EVAL]: {
+    featureFlags: ['disableLMEval'],
+    //TODO: Discuss with the team regarding reliant areas.
+    reliantAreas: [SupportedArea.MODEL_REGISTRY, SupportedArea.MODEL_SERVING],
```

Comment:
I think as per the discussions we have yet, the reliant areas are fine, we can remove this TODO